scenes/menu/main_menu.gd

```diff
@@ -65,6 +67,10 @@ func _ready():
 	_on_char_select_select_character(opponent_selected_character)
 	modal_dialog.visible = false
 	modal_list.visible = false
+	
+	# Initialize settings window
+	settings_popup.visible = false
+	bgm_checkbox.toggled.connect(_on_bgm_check_box_toggled)
```

Comment:
this is the right idea, but I would prefer if PreferencesWindow were the one to emit this message.
PreferencesWindow should listen for the checkbox and then basically re-emit a new signal.